packages/core/src/shared/telemetry/activation.ts

```diff
@@ -110,7 +110,9 @@ function showTelemetryNotice(extensionContext: vscode.ExtensionContext) {
 
     const telemetryNoticeText: string = localize(
         'AWS.telemetry.notificationMessage',
-        'AWS IDE Extensions collects anonymous usage metrics to improve the product. You can opt-out in settings.',
+        `${
+            isAmazonQ() ? 'Amazon Q' : 'AWS Toolkit'
+        } collects anonymous usage metrics to improve the product. You can opt-out in settings.`,
         getIdeProperties().company
```

Comment:
nit: could use `productName` (which takes care of China branding) https://github.com/aws/aws-toolkit-vscode/blob/1b217e978fb269424738c821d91f70d0b9ebaa5d/packages/core/src/shared/extensionUtilities.ts#L40

Also, `getIdeProperties().company` doesn't look like it's used here anymore

Reply:
p.s. technically, `localize()` needs a constant string. to string-replace stuff in a `localize()` string, use a `{0}` token and pass a parameter.

This would matter if/when we get translations for strings.

Reply:
fixed, interesting point about the constant string. Didn't know about that 👍